vllm/v1/metrics/stats.py

```diff
@@ -43,6 +45,7 @@ class SchedulerStats:
         default_factory=PrefixCacheStats)
 
     spec_decoding_stats: Optional[SpecDecodingStats] = None
+    kv_transfer_stats: Optional[KVTransferStatsType] = None
```

Comment:
This is currently the "blocker" which is preventing OOT connectors from implementing their own Stats object, as it needs to msgpacked for sending.

I am planning to address that in a separate PR as it will likely need adding registration + sending an plain dictionary to reconstruct dynamically

cc @sdavidbd